core/opendaq/opendaq/tests/test_core_events.cpp

```diff
@@ -1376,3 +1376,57 @@ TEST_F(CoreEventTest, StatusChangedMuted)
 
     ASSERT_EQ(changeCount, 1);
 }
+
```

Comment:
I'd also include testing of objects stored under the `Type` and `TypeName` keys from event arguments, in addition to testing event IDs.